definitions/npm/rxjs_v5.0.x/flow_v0.29.0-/rxjs_v5.0.x.js

```diff
@@ -166,18 +231,32 @@ declare class rxjs$Observable<+T> {
   mapTo<U>(value: U): rxjs$Observable<U>;
 
   merge(other: rxjs$Observable<T>): rxjs$Observable<T>;
+  merge(others: Iterable<rxjs$Observable<T>>): rxjs$Observable<T>;
+  merge(...others: Array<rxjs$Observable<T>>): rxjs$Observable<T>;
+
+  static merge(other: rxjs$Observable<T>): rxjs$Observable<T>;
```

Comment:
I'm pretty sure the existing typedefs for `static merge` are more precise. Could you maintain that precision or add tests to prove that something like `Observable.merge(numbers, strings)` has type `Observable<number | string>`